code/modules/lighting/lighting_process.dm

```diff
@@ -25,4 +25,4 @@
 
 		scheck()
 
-	lighting_update_overlays.Cut()
```

Comment:
Don't do this yet. 

Len = 0 is faster than cut(), but it incurs memory leaks; until this issue is patched, cut() is better (unless you've heard otherwise and it has, in fact been patched?)